src/scheduler/fifo.c

```diff
@@ -2327,3 +2350,357 @@ next_job(status *policy, server_info *sinfo, int flag)
 	}
 	return rjob;
 }
+
+/**
+ * @brief
+ *	Helper function used to copy the given source string to the destination string. It also
+ *	frees the destination and then allocates required memory before copying.
+ *
+ * @param[in] dest - Address of pointer to destination string
+ * @param[in] dest - pointer to source string
+ *
+ * @retval
+ * @return 0 - Failure
+ * @return  1 - Success
+ *
+ * @par Side Effects:
+ *	None
+ *
+ *
+ */
+static int
+copy_attr_value(char **dest, char *src)
```

Comment:
We can not actually make use of pbs_strcat() for this scenario it is because this function expects at least dest is pre-allocated with some memory prior to this call(and also we need to pass the initial size of dest to this function) which is not true in our case. Also as per Bhroam's earlier comment I have already moved this function to misc.c